### ISSUE TITLE
reset() doesn't show progress bar on reuse

### ISSUE THREAD

**Issue by mprannoy** [tqdm/tqdm#745]
When I use `reset` for reusing the tqdm object created, it doesn't show the progress bar the second time. This is because the `disable` flag is being set to True in `close()`.
  ```python
  a = range(100)
a = tqdm(a)
for i in a:
    pass
a.reset()
for i in a:
    pass
  ```
Can we add `self.disable = False` in `reset`?

**Comment by casperdcl** [comment]
yes we'd need to document inheriting with a close function that does nothing (potentially adding this as a separate class in a subpackage).

something like:

```
from tqdm import tqdm as core_tqdm

class tqdm(core_tqdm):
  def close(self):
    self.refresh()
```

**Comment by mprannoy** [comment]
Won't that break other places where `close` is called? I was thinking we store how `disable` is initialized as `init_disable` and change `disable` to `False` in `reset` if `init_disable` is False.
